libfwupd/fwupd-bios-attr.c

```diff
@@ -496,6 +496,7 @@ fwupd_bios_attr_set_current_value(FwupdBiosAttr *self, const gchar *value)
 /**
  * fwupd_bios_attr_to_variant:
  * @self: a #FwupdBiosAttr
+ * @trusted: whether the caller should receive trusted values
```

Comment:
we used FwupdDeviceFlags for this in other places, but maybe that's wrong here. What you have is probably fine.